src/main/java/SYSC6/Project/Main_Controller.java

```diff
@@ -232,7 +299,39 @@ public User getUser(Long id){
             }
         }
         catch (IOException e){
-            System.out.println("Error");
+            System.out.println("getUser() Error");
+        }
+        return user;
+    }
+
+    public Admin getAdmin(Long id){
```

Comment:
Although moe prefers calls to the rest methods for fetching user data and updating use data, this go around method derived from my original method at least performs the call even if done through Java